build/x86_64_sles12/Dockerfile

```diff
@@ -28,9 +28,6 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
  && /bin/sed -i -e 's/VERSION = 42.3/VERSION = 12/' /etc/SuSE-release \
```

Comment:
Let's add the same comment here as in the logging agent repo: `# Pretend we're on SLES 12.`.